### ISSUE TITLE
Windows 10 Installation fails (probably dependencies misconfiguration)

### ISSUE THREAD

**Issue by ZenaMel** [lektor/lektor#914]
Steps to reproduce:

1. open `cmd`
2. run ```@powershell -NoProfile -Command "(new-object net.webclient).DownloadString('https://www.getlektor.com/installer.py') | python"``` as instructed [here](https://www.getlektor.com/docs/installation/)
3. observe stacktrace:

```
Installing virtual environment...
Collecting Lektor
  Using cached Lektor-3.2.0-py2.py3-none-any.whl (2.1 MB)
Collecting Jinja2>=2.11
  Using cached Jinja2-3.0.1-py3-none-any.whl (133 kB)
Collecting EXIFRead
  Using cached ExifRead-2.3.2-py3-none-any.whl (38 kB)
Collecting requests[security]
  Using cached requests-2.25.1-py2.py3-none-any.whl (61 kB)
Collecting click>=6.0
  Using cached click-8.0.1-py3-none-any.whl (97 kB)
Collecting Werkzeug<1
  Using cached Werkzeug-0.16.1-py2.py3-none-any.whl (327 kB)
Requirement already satisfied, skipping upgrade: pip in c:\users\studium\appdata\local\lektor-cli\lib\lib\site-packages (from Lektor) (20.2.1)
Collecting watchdog
  Using cached watchdog-2.1.3-py3-none-win32.whl (75 kB)
Collecting inifile
  Using cached inifile-0.4.zip (7.4 kB)
Requirement already satisfied, skipping upgrade: setuptools in c:\users\studium\appdata\local\lektor-cli\lib\lib\site-packages (from Lektor) (49.2.1)
Collecting filetype>=1.0.7
  Using cached filetype-1.0.7-py2.py3-none-any.whl (15 kB)
Collecting Flask
  Using cached Flask-2.0.1-py3-none-any.whl (94 kB)
Collecting python-slugify
  Using cached python_slugify-5.0.2-py2.py3-none-any.whl (6.7 kB)
Collecting mistune<2,>=0.7.0
  Using cached mistune-0.8.4-py2.py3-none-any.whl (16 kB)
Collecting Babel
  Using cached Babel-2.9.1-py2.py3-none-any.whl (8.8 MB)
Collecting MarkupSafe>=2.0
  Using cached MarkupSafe-2.0.1-cp38-cp38-win32.whl (14 kB)
Collecting certifi>=2017.4.17
  Using cached certifi-2021.5.30-py2.py3-none-any.whl (145 kB)
Collecting idna<3,>=2.5
  Using cached idna-2.10-py2.py3-none-any.whl (58 kB)
Collecting chardet<5,>=3.0.2
  Using cached chardet-4.0.0-py2.py3-none-any.whl (178 kB)
Collecting urllib3<1.27,>=1.21.1
  Using cached urllib3-1.26.6-py2.py3-none-any.whl (138 kB)
Collecting cryptography>=1.3.4; extra == "security"
  Using cached cryptography-3.4.7-cp36-abi3-win32.whl (1.4 MB)
Collecting pyOpenSSL>=0.14; extra == "security"
  Using cached pyOpenSSL-20.0.1-py2.py3-none-any.whl (54 kB)
Collecting colorama; platform_system == "Windows"
  Using cached colorama-0.4.4-py2.py3-none-any.whl (16 kB)
Collecting itsdangerous>=2.0
  Using cached itsdangerous-2.0.1-py3-none-any.whl (18 kB)
Collecting text-unidecode>=1.3
  Using cached text_unidecode-1.3-py2.py3-none-any.whl (78 kB)
Collecting pytz>=2015.7
  Using cached pytz-2021.1-py2.py3-none-any.whl (510 kB)
Collecting cffi>=1.12
  Using cached cffi-1.14.5-cp38-cp38-win32.whl (167 kB)
Collecting six>=1.5.2
  Using cached six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting pycparser
  Using cached pycparser-2.20-py2.py3-none-any.whl (112 kB)
Using legacy 'setup.py install' for inifile, since package 'wheel' is not installed.
Installing collected packages: MarkupSafe, Jinja2, EXIFRead, certifi, idna, chardet, urllib3, pycparser, cffi, cryptography, six, pyOpenSSL, requests, colorama, click, Werkzeug, watchdog, inifile, filetype, itsdangerous, Flask, text-unidecode, python-slugify, mistune, pytz, Babel, Lektor
    Running setup.py install for inifile ... done
ERROR: After October 2020 you may experience errors when installing or updating packages. This is because pip will change the way that it resolves dependency conflicts.

We recommend you use --use-feature=2020-resolver to test your packages with the new resolver before it becomes the default.

flask 2.0.1 requires Werkzeug>=2.0, but you'll have werkzeug 0.16.1 which is incompatible.
Successfully installed Babel-2.9.1 EXIFRead-2.3.2 Flask-2.0.1 Jinja2-3.0.1 Lektor-3.2.0 MarkupSafe-2.0.1 Werkzeug-0.16.1 certifi-2021.5.30 cffi-1.14.5 chardet-4.0.0 click-8.0.1 colorama-0.4.4 cryptography-3.4.7 filetype-1.0.7 idna-2.10 inifile-0.4 itsdangerous-2.0.1 mistune-0.8.4 pyOpenSSL-20.0.1 pycparser-2.20 python-slugify-5.0.2 pytz-2021.1 requests-2.25.1 six-1.16.0 text-unidecode-1.3 urllib3-1.26.6 watchdog-2.1.3
WARNING: You are using pip version 20.2.1; however, version 21.1.3 is available.
```

**Comment by ZenaMel** [comment]
I've tried upgrading my Werkzeug version via `pip` but since Lektor utilizes its own virtual environment that didn't help.

**Comment by ZenaMel** [comment]
`pip show Werkzeug` yields:

```
Name: Werkzeug
Version: 2.0.1
Summary: The comprehensive WSGI web application library.
Home-page: https://palletsprojects.com/p/werkzeug/
Author: Armin Ronacher
Author-email: armin.ronacher@active-4.com
License: BSD-3-Clause
Location: OBSCURED
Requires:
Required-by: Flask
```

**Comment by ZenaMel** [comment]
My guess is that this line is at fault: https://github.com/lektor/lektor/blob/f273c6918b5f1043a6b09980de0a82f758033929/setup.py#L47

**Comment by andoresuperesu** [comment]
Duplicate of #910

**Comment by yagebu** [comment]
This should be fixed by #911
